.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=1234
+MYSQLDB_DATABASE=online_book_store
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
You haven't to push it to Git Hub. You may delete the file, push, and add it again to a local folder. Maybe you added ".env" to .gitignore after git detects this file.

Reply:
Also, you may use these variables in application.properites using ${var_name}